locations/spiders/jdsports_gb.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+from locations.linked_data_parser import LinkedDataParser
+
+
+class JDSportsGBSpider(CrawlSpider):
+    name = "jdsports_gb"
+    item_attributes = {
+        "brand": "JD Sports",
+        "brand_wikidata": "Q6108019",
+    }
+    start_urls = ["https://www.jdsports.co.uk/store-locator/all-stores/"]
+    rules = [
+        Rule(
+            LinkExtractor(allow="store-locator/"), callback="parse_store", follow=False
+        )
+    ]
+    download_delay = 0.2
+
+    def parse_store(self, response):
+        if "-soon" in response.url:
```

Comment:
Can this be done as a 'deny' in the LinkExtractor? That way we'd skip the http request.

Reply:
There's just 2 of them. It's clearer in code I think.